### PR TITLE
[Chore] - Strict Template and client tests

### PR DESCRIPTION
**Pull request by FuchsDominik**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple integration tests (Jest) related to the features (with a high test coverage)

### Motivation and Context

client tests and strict template fixes

### Description

strict template fixes for health component and client tests for manual text selection

### Test Coverage

manual-text-selection.component.ts: 100%

### CODE REVIEW

File: src/main/webapp/app/admin/health/health.component.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 
 import { Health, HealthDetails, HealthKey, HealthService } from './health.service';
```
Comment (FrankeLukas) [inline src/main/webapp/app/admin/health/health.component.ts:5]: HealthKey is unused now.

File: src/test/javascript/spec/component/text/manual-text-selection.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    });
+
+    it('should select text and assess the text', () => {
+        const rectangle = { left: 0, top: 0, width: 0, height: 0 } as SelectionRectangle;
+        const event = { text: 'This is a text\n another line', viewportRectangle: null, hostRectangle: null } as TextSelectEvent;
+
+        component.disabled = true;
+
+        // catch edge case
+        component.didSelectSolutionText(event);
```
Comment (FrankeLukas) [inline src/test/javascript/spec/component/text/manual-text-selection.component.spec.ts:50]: Is this missing an `expect`? (We overwrite the values immediately in the next lines.)

**Review by FrankeLukas — APPROVED** [review]
Template changes lgtm.

File: src/main/webapp/app/admin/health/health.component.ts (hunk 1 above)
Comment (FuchsDominik) [reply src/main/webapp/app/admin/health/health.component.ts]: 👍

File: src/test/javascript/spec/component/text/manual-text-selection.component.spec.ts (hunk 2 above)
Comment (FuchsDominik) [reply src/test/javascript/spec/component/text/manual-text-selection.component.spec.ts:50]: To achieve 100% coverage, i wanted to cover the edge case when disabled is set to true. It does not set any data so nothing can be expected here

**Review by iozbeyli — APPROVED** [review]
Code looks good

File: src/test/javascript/spec/component/text/manual-text-selection.component.spec.ts (hunk 2 above)
Comment (FrankeLukas) [reply src/test/javascript/spec/component/text/manual-text-selection.component.spec.ts:50]: Ah ok.

**Review by FrankeLukas — APPROVED** [review]

File: src/main/webapp/app/admin/health/health-modal.component.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export class HealthModalComponent {
 
     constructor(public activeModal: NgbActiveModal) {}
 
-    readableValue(value: number): string {
+    readableValue(value: any): string {
```
Comment (krusche) [inline src/main/webapp/app/admin/health/health-modal.component.ts:15]: is this change really needed? we would like to avoid any as type

Comment (FuchsDominik) [reply src/main/webapp/app/admin/health/health-modal.component.ts:15]: I cannot really assure that `value` is a number here. `readableValue()` is used only once with a very nested attribute called `health.value.details[].value` where details is of type any. I could declare details as a { key, value } object but as the test instantiate this value with only a string, i decided i will use this safer method and declare this parameter as any instead.
